src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -457,6 +468,8 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration)
             // Start page should not show up during test mode.
             this.ShowStartPage = !DynamoModel.IsTestMode;
 
+            this.BrandingResourceProvider = resourceProvider ?? new DefaultBrandingResourceProvider();
+
```

Comment:
I thought the same but the `Start` method is static and I can't assign a property inside it.